tracing/src/main/java/io/airlift/tracing/OpenTelemetryConfig.java

```diff
@@ -22,4 +28,22 @@ public OpenTelemetryConfig setEndpoint(String endpoint)
         this.endpoint = endpoint;
         return this;
     }
+
+    @NotNull
+    public Map<String, String> getResourceAttributes()
+    {
+        return resourceAttributes;
+    }
+
+    @Config("tracing.resource.attributes")
+    public OpenTelemetryConfig setResourceAttributes(String resourceAttributes)
```

Comment:
You should take a look at how we allow setting additional annotations in logging in the logmanager. We allow for setting a separate property file with key-value pairs. It's a little cleaner.

Reply:
Actually, it would be great if we refactored that for this so we just set it up once.

Reply:
Done! We'll probably be able to reuse the same properties file here, this is better, thanks!

I'm not sure what refactoring you mean, this is already called from a singleton provider.

Reply:
Yeah, if you want to rename that one to be something more generic.